tests/cartToCheckout.spec.mjs

```diff
@@ -0,0 +1,69 @@
+import { test, expect } from '@playwright/test';
+import dotenv from "dotenv";
+import productModel from '../models/productModel';
+import mongoose from 'mongoose';
+
+dotenv.config();
+
+test.beforeAll( async () => {
+    await mongoose.connect(process.env.MONGO_URL);
+
+    const book = new productModel({
+        name: 'best-book',
+        slug: 'book-slug',
+        description: 'this is the best book',
+        price: 18,
+        category: new mongoose.Types.ObjectId('64b0c0f9a4b1a33d8e4a0d0e'),
+        quantity: 1,
+        photo: {
+            data: Buffer.from('sample photo data', 'utf-8'),
+            contentType: 'image/jpeg',
+        },
+        shipping: false,
+    });
+
+    const laptop = new productModel({
+        name: 'best-laptop',
+        slug: 'laptop-slug',
+        description: 'this is the best laptop',
+        price: 1500,
+        category: new mongoose.Types.ObjectId('74b0c0f9a4b1a33d8e4a0d0e'),
+        quantity: 1,
+        photo: {
+            data: Buffer.from('sample photo data', 'utf-8'),
+            contentType: 'image/jpeg',
+        },
+        shipping: false,
+    });
+
+    await book.save();
+    await laptop.save();
+});
+
+test.afterAll( async () => {
+    await productModel.deleteOne({ slug: 'book-slug' });
+    await productModel.deleteOne({ slug: 'laptop-slug' });
+    mongoose.disconnect();
+});
+
+test('add to cart and checkout', async ({ page }) => {
```

Comment:
I tried this test and it indeed passes. However, I remember that when I was trying the cart, the price for each item shown in the cart page is not correctly displayed to 2 decimal places. You are not facing this issue now as your items have prices that are integers. Not sure if you want to consider such a case, just for your reference.

For example:
![image](https://github.com/user-attachments/assets/9d62db84-a848-4330-b6cd-7530ec4d30fa)


Your case:
![image](https://github.com/user-attachments/assets/b1d300a8-b114-41f0-a8b0-b42583ae3622)

Reply:
Ohh, I was mainly checking for the total price to be correct so I guess I will just leave it, thanks for highlighting though 😃